modules/miscutil/lib/orcidutils.py

```diff
@@ -777,7 +793,8 @@ def _stub_method():
                     _stub_method()
                 except DoubledIds:
                     register_exception(subject="The paper %s contains the"
-                                       "same ArXiv id %s twice." % (recid, encoded),
+                                       "same ArXiv id %s twice." % (recid,
+                                                                    encoded),
```

Comment:
Actually it's the fact that the `encoded` variable is `unicode` the culprit of the not correctly handled email headers.